<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolarSystemWidget</class>
 <widget class="QWidget" name="SolarSystemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1772</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="topPanel" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhUppercaseOnly</set>
     </property>
     <layout class="QHBoxLayout" name="topPanelLayout">
      <item>
       <widget class="QLabel" name="lbl_meshName">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="modelNameLabel">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="inputMethodHints">
         <set>Qt::ImhUppercaseOnly</set>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="progressLabel">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="format">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbLoadAsteroid">
        <property name="inputMethodHints">
         <set>Qt::ImhUppercaseOnly</set>
        </property>
        <property name="text">
         <string>Load Model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomPanel" native="true">
     <property name="inputMethodHints">
      <set>Qt::ImhUppercaseOnly</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="inputMethodHints">
         <set>Qt::ImhUppercaseOnly</set>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhUppercaseOnly</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QWidget" name="currentPositionWidget" native="true">
            <layout class="QHBoxLayout" name="positionLayout"/>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="orbitWidget" native="true">
            <layout class="QHBoxLayout" name="orbitLayout"/>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Epoch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Year</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_year">
                  <property name="text">
                   <string>2018</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Month</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_month">
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Day</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_day">
                  <property name="text">
                   <string>10</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_31">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Object Orbit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_14" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Preset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmb_presetOrbit">
                  <item>
                   <property name="text">
                    <string>Mars</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_12" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_28">
                  <property name="text">
                   <string>Light Intensity</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_lightIntensity">
                  <property name="text">
                   <string>50</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Angular Velocity (degree/second)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>x</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_wX">
                  <property name="text">
                   <string>10</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_wY">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>z</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_wZ">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_32">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Camera Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_16" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>yfov</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_cameraYfov">
                  <property name="text">
                   <string>1.04</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>aspect ratio</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_cameraAspectRatio">
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_25">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Simulation Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_11" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Period (s)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_period">
                  <property name="text">
                   <string>2</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_13" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>Dt (s)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_dt">
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_15" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>Shutter Speed (s)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ln_shutterSpeed">
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_10" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pb_play">
                  <property name="text">
                   <string>Play</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_stop">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Stop</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="widget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhUppercaseOnly</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="mapAndPlotPanel" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhUppercaseOnly</set>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QWidget" name="renderedImageWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>350</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhUppercaseOnly</set>
               </property>
               <layout class="QVBoxLayout" name="renderedImageLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="lineCurveWidget" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>350</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhUppercaseOnly</set>
               </property>
               <layout class="QHBoxLayout" name="lineCurveLayout"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
